src/client.rs

```diff
@@ -194,6 +269,58 @@ impl DatadogTracing {
             .map(|_| ())
             .map_err(|_| ())
     }
+
+    pub fn send_log(&self, record: LogRecord) -> Result<(), ()> {
+        self.buffer_sender
+            .lock()
+            .unwrap()
+            .send(SpanState::Log(record))
+            .map(|_| ())
+            .map_err(|_| ())
+    }
+}
+
+impl Log for DatadogTracing {
+    /// Determines if a log message with the specified metadata would be
```

Comment:
One line? As an aside: http://lkml.iu.edu/hypermail/linux/kernel/2005.3/08168.html

Reply:
I don't understand this one, but I'll remove these comments anyway as they come from the trait def itself.

Reply:
Sorry, what I meant was that the one word, I think it was `logger`, that was on the next line could probably be moved up.